Tests/BasicTests/CollectionAlgorithmsTests.swift

```diff
@@ -13,15 +13,6 @@ import XCTest
 import Basic
 
 class CollectionAlgorithmsTests: XCTestCase {
-    func testRIndex() {
```

Comment:
We also need to remove the entry for this test in `Tests/BasicTests/XCTestManifests.swift`.

Reply:
Done and pushed!